context-core/src/main/java/de/atb/context/common/util/BusinessCase.java

```diff
@@ -35,7 +35,16 @@
 	private String id;
 	private String url;
 
-	public static void getInstance() {
+    public BusinessCase() {
```

Comment:
Is `BusinessCase` supposed to be a Singleton? If so, shoulnd't this constructor be private?

Reply:
I would suggest to not do this change in this Merge Request, but consider it for improvement in a later dev branch

Reply:
done